Command/CreateClientCommand.php

```diff
@@ -67,13 +69,28 @@ protected function configure(): void
                 InputArgument::OPTIONAL,
                 'The client secret'
             )
+            ->addOption(
+                'public',
+                null,
```

Comment:
@X-Coder264 This seems like a thing that could be used often, maybe we should add a shortcut?

Reply:
@HypeMC We don't provide shortcuts for any of the options in this command and I don't think we should provide a shortcut for creating a public client - it's always best to be explicit when dealing with such security related stuff.